### PR TITLE
Fix: auth not working for me, database:init script not working, resolve package-lock 

### PR DESCRIPTION
**Pull request by Roman-Octavian**
- Auth was not actually setting any cookies (for me at least)
- `package-lock` was disturbed in #6 ; I noticed the diff was too large but didn't have time to look into it. When I tried to run the app now I got missing packages error.
- `npm run database:init` actually works now.
- `npm run database:init` sets user pass `admin` `admin` to simplify login during development, default image
- image storage functions include cookies, otherwise auth requirements will make them fail

### CODE REVIEW

**Comment by brrobert17** [comment]
I successfully tested it. Although Im a little puzzled why it didn't send the cookie because you correctly included the credentials with your fetches... anyways I hope it works for you now as well

**Review by brrobert17 — APPROVED** [review]
all good

**Comment by nikidkny** [comment]
image

For some reason the token is stuck in the cookies for me

**Comment by Roman-Octavian** [comment]
> image
> 
> For some reason the token is stuck in the cookies for me, I haven't tried to log in since yesterday evening and it is still there

@nikidkny I highly doubt this is going to handle cookies all that well. Install this extension:

[Chrome](https://chromewebstore.google.com/detail/cookie-editor/hlkenndednhfkekhgcdicdfddnkalmdm)
[Firefox](https://addons.mozilla.org/en-US/firefox/addon/cookie-editor/)

And wipe them yourself if you have trouble. Just use a private tab for new logins

File: package-lock.json [hunk 1]
```diff
@@ -37,9 +37,8 @@
     },
     "node_modules/@alloc/quick-lru": {
       "version": "5.2.0",
-      "resolved": "https://registry.npmjs.org/@alloc/quick-lru/-/quick-lru-5.2.0.tgz",
-      "integrity": "sha512-UrcABB+4bUrFABwbluTIBErXwvbsU/V7TZWfmbgJfbkwiBuziS9gxdODUyuiecfdGQ85jglMW6juS3+z5TsKLw==",
       "dev": true,
+      "license": "MIT",
```
[the remark this answers is not in the document]
Comment (nikidkny) [reply package-lock.json]: image

Yeah, just a couple of warnings

[the remark this answers is not in the document]
Comment (Roman-Octavian) [reply package-lock.json]: Can you run the app with `npm run dev` ?

[the remark this answers is not in the document]
Comment (nikidkny) [reply package-lock.json]: yes

[the remark this answers is not in the document]
Comment (Roman-Octavian) [reply package-lock.json]: Ok then it's @brrobert17 that's somehow breaking the package lock when he installs

File: client/storage/index.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { type UploadResponse, type DeleteResponse } from '@packages/types';
+import { axiosInstance } from '../utils/axiosConfig.ts';
```
[the remark this answers is not in the document]
Comment (brrobert17) [reply client/storage/index.ts]: That's the very reason Im using axios is that Im using an interceptor to refresh the token

[the remark this answers is not in the document]
Comment (brrobert17) [reply client/storage/index.ts]: I did't realise there was a simpler way of implementing that

File: client/storage/index.ts [hunk 3]
```diff
@@ -12,25 +13,17 @@ export async function uploadImage(file: File): Promise<UploadResponse> {
     const formData = new FormData();
     formData.append('file', file);
 
-    const response = await fetch('http://localhost:8080/api/v1/storage/upload', {
-      method: 'POST',
-      credentials: 'include',
-      body: formData,
+    const response = await axiosInstance.post<UploadResponse>('/storage/upload', formData, {
```
[the remark this answers is not in the document]
Comment (brrobert17) [reply client/storage/index.ts]: Im sorry Im aware of git conventions on such a deep level, to me it seemed like the fix branch was created as a request for me to fix what is my responsibility

**Comment by brrobert17** [comment]
The refresh token is set to stay alive for a day. So when I'm testing I delete them manually. If you need it there's a logout endpoint that is supposed to clear all your cookies

File: package-lock.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Roman-Octavian) [reply package-lock.json]: @brrobert17 try to do `npm i` at root on this branch, then do `npm run dev`. Let us know if it works

[the remark this answers is not in the document]
Comment (brrobert17) [reply package-lock.json]: it still works just fine

[the remark this answers is not in the document]
Comment (Roman-Octavian) [reply package-lock.json]: does it add a bunch of changes to `package-lock.json` ?

[the remark this answers is not in the document]
Comment (brrobert17) [reply package-lock.json]: no. Why dont you just delete the package-lock and reinstall ?

[the remark this answers is not in the document]
Comment (Roman-Octavian) [reply package-lock.json]: I don't need to. I reverted your commit and reinstalled already. If it works for you too then the issue I guess was you installing in all directories separately that was causing issues.

**Comment by brrobert17** [comment]
Why did you take out the path from the refresh cookie? That cookie only needs to go to the /refresh endpoint to get a new access token

**Comment by brrobert17** [comment]
Alright. But if you reverted my changes and you're not fetching in the storage-example with axios, then the interceptor will be ignored and you wont refresh your token automatically

**Pull request merged**